--- vlock-2.2.2/src/vlock-main.c.orig	2008-05-18 11:10:54.000000000 +0200
+++ vlock-2.2.2/src/vlock-main.c	2008-05-31 14:04:33.852180049 +0200
@@ -172,7 +172,8 @@
       sleep(1);
 
 #ifndef NO_ROOT_PASS
-    if (strcmp(username, "root") != 0) {
+    if ((strcmp(username, "root") != 0)
+	&& (geteuid() == 0)) { /* checking root password with PAM needs suid-root */
       /* Try authentication as root. */
       if (auth("root", prompt_timeout))
         break;
