--- vlock-2.2.2/modules/Makefile.orig	2008-05-18 11:10:54.000000000 +0200
+++ vlock-2.2.2/modules/Makefile	2008-05-31 14:08:18.652978146 +0200
@@ -37,7 +37,7 @@
 .PHONY: install-%.so
 install-%.so: %.so
 	$(MKDIR_P) -m 755 $(DESTDIR)$(MODULEDIR)
-	$(INSTALL) -m $(MODULE_MODE) -o root -g $(MODULE_GROUP) $< $(DESTDIR)$(MODULEDIR)/$<
+	$(INSTALL) -m $(MODULE_MODE) $< $(DESTDIR)$(MODULEDIR)/$<
 
 .PHONY: clean
 clean:
--- vlock-2.2.2/Makefile.orig	2008-05-18 11:10:54.000000000 +0200
+++ vlock-2.2.2/Makefile	2008-05-31 14:07:55.195641388 +0200
@@ -57,9 +57,9 @@
 .PHONY: install-programs
 install-programs: $(PROGRAMS)
 	$(MKDIR_P) -m 755 $(DESTDIR)$(PREFIX)/bin
-	$(INSTALL) -m 755 -o root -g $(ROOT_GROUP) vlock $(DESTDIR)$(BINDIR)/vlock
+	$(INSTALL) -m 755 vlock $(DESTDIR)$(BINDIR)/vlock
 	$(MKDIR_P) -m 755 $(DESTDIR)$(PREFIX)/sbin
-	$(INSTALL) -m 4711 -o root -g $(ROOT_GROUP) vlock-main $(DESTDIR)$(SBINDIR)/vlock-main
+	$(INSTALL) -m 4755 vlock-main $(DESTDIR)$(SBINDIR)/vlock-main
 
 .PHONY: install-plugins
 install-plugins: install-modules install-scripts
@@ -75,11 +75,11 @@
 .PHONY: install-man
 install-man:
 	$(MKDIR_P) -m 755 $(DESTDIR)$(MANDIR)/man1
-	$(INSTALL) -m 644 -o root -g $(ROOT_GROUP) man/vlock.1 $(DESTDIR)$(MANDIR)/man1/vlock.1
+	$(INSTALL) -m 644 man/vlock.1 $(DESTDIR)$(MANDIR)/man1/vlock.1
 	$(MKDIR_P) -m 755 $(DESTDIR)$(MANDIR)/man8
-	$(INSTALL) -m 644 -o root -g $(ROOT_GROUP) man/vlock-main.8 $(DESTDIR)$(MANDIR)/man8/vlock-main.8
+	$(INSTALL) -m 644 man/vlock-main.8 $(DESTDIR)$(MANDIR)/man8/vlock-main.8
 	$(MKDIR_P) -m 755 $(DESTDIR)$(MANDIR)/man5
-	$(INSTALL) -m 644 -o root -g $(ROOT_GROUP) man/vlock-plugins.5 $(DESTDIR)$(MANDIR)/man5/vlock-plugins.5
+	$(INSTALL) -m 644 man/vlock-plugins.5 $(DESTDIR)$(MANDIR)/man5/vlock-plugins.5
 
 
 ### build rules ###
--- vlock-2.2.2/scripts/Makefile.orig	2008-05-18 11:10:54.000000000 +0200
+++ vlock-2.2.2/scripts/Makefile	2008-05-31 14:53:25.275223109 +0200
@@ -12,7 +12,7 @@
 install-%.sh : SCRIPT_TARGET=$(<:.sh=)
 install-%: %
 	$(MKDIR_P) -m 755 $(DESTDIR)$(SCRIPTDIR)
-	$(INSTALL) -m $(SCRIPT_MODE) -o root -g $(SCRIPT_GROUP) $< $(DESTDIR)$(SCRIPTDIR)/$(SCRIPT_TARGET)
+	$(INSTALL) -m $(SCRIPT_MODE) $< $(DESTDIR)$(SCRIPTDIR)/$(SCRIPT_TARGET)
 
 .PHONY: clean
 clean:
